package.json

```diff
@@ -48,7 +49,7 @@
     "react-native-svg": "^12.1.1",
     "react-native-svg-transformer": "^0.14.3",
     "react-native-tts": "4.1.0",
-    "react-native-vector-icons": "^9.0.0",
+    "react-native-vector-icons": "^9.1.0",
```

Comment:
please undo

Reply:
If we do it in a separate ticket, Martin will know and can test, when we do it here, we now have to do research, if it breaks something and hope everything is fine. If there is no need, I would not do it here. @klinzo any opinions on this?

Reply:
Agreeing with @SteffiStuffel , upgrades should only be done if there is a reason for it or if there is a task for it.